tests/test_feature_selection.py

```diff
@@ -0,0 +1,101 @@
+# Authors: Soledad Galli <solegalli1@gmail.com>
+# License: BSD 3 clause
+import pytest
+import pandas as pd
+
```

Comment:
according to pep8, imports are done alphabetically, so first pandas then pytest. Could you re-order?